client/settings.json

```diff
@@ -2,5 +2,6 @@
 	"driverPort1": "COM5",
 	"driverPort2": "COM4", 
 	"videoDir": "",
-	"imageDir": ""
+	"imageDir": "",
+	"IPSERVER": ""

```

Comment:
Can we change this to ipServer, or something so that the casing of all settings is consistent? Not that there's anything inherently wrong with this, but just inconsistent with the existing settings

Reply:
Agreed & fixed